src/plugins/kibana/public/settings/sections/indices/directives/install_filebeat_step.html

```diff
@@ -0,0 +1,34 @@
+<div class="wizard-step-title">
+  <h3>Follow these instructions to install Filebeat</h3>
+  Now that you've got a fresh pipeline and index pattern, let's throw some data at it!
```

Comment:
Yeah, this is an interesting question. I think in an ideal world, the user would never have to know anything about how the pipeline actually gets persisted to Elasticsearch. But given that we won't have the ability to manage pipelines right off the bat, advanced users will probably benefit from this kind of information. Maybe we could have a collapsed by default section on this page that provides additional info like this?